GPU/GPUTracking/DataTypes/GPUTRDTrack.h

```diff
@@ -40,10 +40,10 @@ namespace gpu
 {
 template <typename T>
 class GPUTRDTrack_t;
-}
+} // namespace gpu
 } // namespace GPUCA_NAMESPACE
 #else
-#include "GPUTRDInterfaces.h"
+#include "GPUTRDInterfaceO2Track.h"
```

Comment:
@davidrohr could you add a protection here to not include that header for AliRoot please? I just tried to compile AliRoot, because when I change the `GPUTRDTrack` a lot I want to make sure that the tracking with AliRoot still works